tests/foreman/cli/test_contentviews.py

```diff
@@ -250,6 +250,44 @@ def test_cv_delete(self):
         with self.assertRaises(CLIReturnCodeError):
             ContentView.info({'id': con_view['id']})
 
+    def test_delete_cv_files(self):
+        """@Test: Delete content view and verify it was actually deleted from
+        hard drive.
+
+        @Feature: Content View
+
+        @Assert: Content view was deleted and pulp folder doesn't contain
+        content view files anymore
+
+        """
+        # Create a content view and publish it
+        content_view = make_content_view({'organization-id': self.org['id']})
```

Comment:
Not sure if there is a real need for second test (as this test is a dummy test without repo IMO).  I vote for updating this test with real repo, synchronize, add it to content view before publishing.